ngi_visualizations/fpkm_scatter/fpkm_scatter.py

```diff
@@ -213,13 +255,98 @@ def plot_fpkm_scatter (sample_1, sample_2, x_lab, y_lab, output_fn=False, linear
     logging.info("Saving to {} and {}".format(png_fn, pdf_fn))
     plt.savefig(png_fn)
     plt.savefig(pdf_fn)
+    
+    # Return the filenames and R2
+    return {'png': png_fn, 'pdf': pdf_fn, output_fn:r_squared }
```

Comment:
Nope, it's a variable. But yeah there is a variable with the same name in another function but neither of them are global.